generator/scripts/generateInputRange.py

```diff
@@ -0,0 +1,97 @@
+import openpyxl  # Read data from Excel
+
+
+# Function to replace single quotes and stringify text
+def replaces_quotes_and_stringify(text):
```

Comment:
Do you need this function in other scripts? If so, you can put it in a separate file that you can include in the script here

Reply:
Yes indeed, I use it in another file, so I will put it in another place.